ckanext/ytp/comments/helpers.py

```diff
@@ -27,12 +27,20 @@ def show_comments_tab_page():
 
 
 def profanity_check(cleaned_comment):
-    more_words = load_bad_words()
-    whitelist_words = load_good_words()
+    custom_profanity_list = config.get('ckan.comments.profanity_list', [])
+
+    if custom_profanity_list:
+        pf = ProfanityFilter(custom_censor_list=custom_profanity_list.splitlines())
+    else:
+        # Fall back to original behaviour of built-in Profanity bad words list
+        # combined with bad_words_file and good_words_file
+        more_words = load_bad_words()
```

Comment:
why are you dropping the internal text file list bad_words.txt and good_words.txt.

will this be stored in the database or in the ckan.ini file? 
the current badwords list is over 1600 words/lines long, will this be loaded if it can't find it?

Reply:
@duttonw `ckan.comments.profanity_list` is a run-time configuration option - it will be stored in the database.

The bad words and good words files will be used if the run-time configuration is not set.

There is more context for this in the story, in particular the comments between Carl and myself: 
https://salsadigital.atlassian.net/browse/DQL2-3

Reply:
need info in change log or simliar to state that wordlist text files are not copied into db for editing and is a manual step. we should be adding it to the database if it does not exist (maybe). 
OR
it is extra items to the good list or bad list and is merged with what is in the plugin to save on space etc.

Reply:
@duttonw I believe the aim was to simplify management by maintaining a single list. Currently the plugin has to juggle the library's built-in blacklist plus its own blacklist. This would allow a single master list.

Reply:
may need some info next to that option to state that if blank internal list is used else this list takes precedence.